src/pretix/eventyay_common/views/dashboards.py

```diff
@@ -9,3 +29,124 @@ def organiser_dashboard(request):
         'video_component': '#',
     }
     return render(request, 'eventyay_common/dashboard/dashboard.html', context)
+
+
+def event_index_widgets_lazy(request, organizer, event):
+    subevent = None
+    if request.GET.get("subevent", "") != "" and request.event.has_subevents:
+        i = request.GET.get("subevent", "")
+        with contextlib.suppress(SubEvent.DoesNotExist):
+            subevent = request.event.subevents.get(pk=i)
```

Comment:
Overkill, if the `SubEvent.DoesNotExist` can be ignored and you want to return `None` in that case, the code can be as simple as (without `with` block):

```py
subevent = request.event.subevents.filter(pk=i).first()
```